docs/content/usages/error.md

```diff
@@ -4,70 +4,73 @@ sort: 8
 
 # Error handling
 
-Nikita implements error management by following familiar [Node.js](https://nodejs.org) conventions. The handling of errors different slightly between synchronous and asynchronous functions.
+Nikita's actions throw exceptions in case of failure. When an error occurs the [action promise](/current/usages/promise) is rejected. 
```

Comment:
Actions don't throw errors, they reject errors. Be careful with the wording, actions are promises.

Reply:
> Also show how errors are cascated to the parent with both thrown errors and returned promises.

What do you mean by return promises? Such an example doesn't work by the way:

```js
nikita
// Call parent action
.call(function() {
  // Call child action
  this.call(() => {
    throw Error('Catch me!')
  })
})
// Catch child error
.catch((err) => {
  console.info(err.message) // Catch me!
})
```

You agree with such examples:

```js
nikita
// Call parent action
.call(async function() {
  // Call child action
  await this.call(() => {
    throw Error('Catch me!')
  })
})
// Catch child error
.catch((err) => {
  console.info(err.message) // Catch me!
})
```
and
```js
nikita
// Call parent action
.call(function() {
  // Call child action
  return this.call(() => {
    throw Error('Catch me!')
  })
})
// Catch child error
.catch((err) => {
  console.info(err.message) // Catch me!
})
```

Give me yours in case you are not.